### PR TITLE
Added links

### PR DESCRIPTION
**Pull request by samyakbvs**
I have added 9 links just in case

### CODE REVIEW

**Comment by samyakbvs** [comment]
Issue #123 Fixed

**Comment by samyakbvs** [comment]
@harshit98 Please see if this is fine

File: index.html [hunk 1]
```diff
@@ -2368,9 +2370,8 @@ <h1>Speakers</h1>
 										</div>
 										<div class="social-links vertical-align">
 											<a href="https://www.facebook.com/cocotsaizax"><i class="icon fa fa-external-link"></i></a>
-											<!--<a href="#"><i class="icon fa fa-github"></i></a>
-											<a href="#"><i class="icon social_twitter"></i></a>
-											<a href="#"><i class="icon social_linkedin"></i></a>-->
+											<a href="https://twitter.com/tsaimenghsuan"><i class="icon social_twitter"></i></a>
+											<a href="https://www.linkedin.com/in/menghsuan-tsai-87077273"><i class="icon social_linkedin"></i></a>-->
```
Comment (akshatnitd) [inline index.html]: Remove ```-->```

File: index.html [hunk 2]
```diff
@@ -2804,7 +2809,7 @@ <h1>Speakers</h1>
 											<a href="http://onemakergroup.com"><i class="icon fa fa-external-link"></i></a>
 											<!---<a href=""><i class="icon fa fa-github"></i></a>
 											<a href="#"><i class="icon social_twitter"></i></a>
```
Comment (akshatnitd) [inline index.html]: Close the comments by ```-->```

File: index.html [hunk 3]
```diff
@@ -3139,7 +3146,7 @@ <h1>Speakers</h1>
 										</div>
 										<div class="social-links vertical-align">
 											<a href="https://github.com/AkhilHector"><i class="icon fa fa-github"></i></a>
-											<a href="https://linkedin.com/in/akhilpandey"><i class="icon social_linkedin"></i></a>
+											<a href="https://linkedin.com/in/akhilpandey"><i class="icon social_linkedin"></i></a><a href="https://twitter.com/imakhil_kp "><i class="icon social_twitter"></i></a>
```
Comment (akshatnitd) [inline index.html]: Write the ```<a>``` tags in separate lines for proper readability.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Make the requested changes.

File: index.html [hunk 4]
```diff
@@ -1877,10 +1877,10 @@ <h1>Speakers</h1>
 											It's great to be at FOSSASIA in Singapore! Fantastic atmosphere with lots of friendly devs, designers and business folks.
 										</div>
 										<div class="social-links vertical-align">
-											<a href="http://RedCatLabs.com/"><i class="icon fa fa-external-link"></i></a>
+											<a href="http://RedCatLabs.com/"><i class="icon fa fa-external-link"></i></a>											<a href="https://twitter.com/martinandrews45?lang=en"><i class="icon social_twitter"></i></a>
+
 											<a href="http://github.com/mdda"><i class="icon fa fa-github"></i></a>
-											<!--<a href="#"><i class="icon social_twitter"></i></a>
-											<a href="#"><i class="icon social_linkedin"></i></a>-->
+											<a href="https://www.linkedin.com/in/martinandrews"><i class="icon social_linkedin"></i></a>-->
```
Comment (akshatnitd) [inline index.html]: Remove ```-->``` from the end of <a> tag

**Comment by samyakbvs** [comment]
Done, now is it fine?

File: index.html [hunk 5]
```diff
@@ -2802,9 +2805,9 @@ <h1>Speakers</h1>
 										</div>
 										<div class="social-links vertical-align">
 											<a href="http://onemakergroup.com"><i class="icon fa fa-external-link"></i></a>
-											<!---<a href=""><i class="icon fa fa-github"></i></a>
+											<!---<a href=""><i class="icon fa fa-github"></i></a>-->
```
Comment (akshatnitd) [inline index.html]: The comment closing tag should be after the twitter's <a> tag

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Try to make the changes perfectly this time

**Comment by samyakbvs** [comment]
now is everything fine?

**Review by akshatnitd — APPROVED** [review]
LGTM :rocket: Merging :+1:

**Pull request merged**